examples/kms/vault/vault.yaml

```diff
@@ -100,6 +100,12 @@ items:
             bound_service_account_names="${SERVICE_ACCOUNTS}" \
             bound_service_account_namespaces="${SERVICE_ACCOUNTS_NAMESPACE}" \
             policies="${CLUSTER_IDENTIFIER}"
+
+        # disable iss validation
+        # from: external-secrets/kubernetes-external-secrets#721
+        vault write auth/kubernetes/config \
```

Comment:
`auth/kubernetes/config ` should be `auth/${CLUSTER_IDENTIFIER}/config`?

Reply:
ah, yes, maybe that is it. Minikube may not call the cluster `kubernetes`?

Reply:
it is called kubernetes. https://github.com/ceph/ceph-csi/pull/2203#discussion_r657589265 may be token and ca cert details need to be provided also